test/LockedNORI.test.ts

```diff
@@ -1042,8 +1070,67 @@ describe('LockedNori', () => {
 
       expect(await lNori.balanceOf(employee)).to.equal(newBalance);
       expect(await lNori.totalSupply()).to.eq(newBalance);
+
+      // Ensures grantAmount is set correctly after a revocation
+      const grantDetail = await lNori.getGrant(employee);
+      expect(grantDetail.grantAmount).to.equal(grantAmount.sub(quantityToRevoke));
+      expect(grantDetail.originalAmount).to.equal(grantAmount);
     });
 
+    it('Should revoke a specific amount of unvested tokens repeatedly', async () => {
+        const { lNori, grantAmount, grant } = await setupWithGrant(
+          linearParams()
+        );
+        const { admin } = await hre.getNamedAccounts();
```

Comment:
FYI you can use the following to avoid the `await`

```suggestion
        const { admin } = hre.namedAccounts;
```

Same applies to namedSigners